notebooks/wep_pipeline_timing.ipynb

```diff
@@ -0,0 +1,228 @@
+{
```

Comment:
<p>maybe "#comcam detectors" and maybe include a comment saying, you will need to select the exposures and detectors used by the pipeline run</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>